plugins/georeplication/store-utils.go

```diff
@@ -42,7 +42,7 @@ func getSession(masterid string, remoteid string) (*georepapi.GeorepSession, err
 func addOrUpdateSession(v *georepapi.GeorepSession) error {
 	json, e := json.Marshal(v)
 	if e != nil {
-		log.WithField("error", e).Error("Failed to marshal the Info object")
+		log.WithError(e).Error("Failed to marshal the Info object")
```

Comment:
You have two commits which does error related fixed. Merge all of them into single commit

Reply:
done